doc/metrics.md

```diff
@@ -29,26 +29,59 @@ To view the raw data that LXD collects, use the [`lxc query`](lxc_query.md) comm
+lxd_api_requests_completed_total{entity_type="project",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="project",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="network",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="network",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="network",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="warning",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="warning",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="warning",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="profile",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="profile",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="profile",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="instance",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="instance",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="instance",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="identity",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="identity",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="identity",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="node",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="node",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="node",result="succeeded"} 0
+lxd_api_requests_completed_total{entity_type="operation",result="error_server"} 0
+lxd_api_requests_completed_total{entity_type="operation",result="error_client"} 0
+lxd_api_requests_completed_total{entity_type="operation",result="succeeded"} 0
+# HELP lxd_api_requests_ongoing The number of API requests currently being handled.
+# TYPE lxd_api_requests_ongoing gauge
+lxd_api_requests_ongoing{entity_type="server"} 1
+lxd_api_requests_ongoing{entity_type="storage_pool"} 0
+lxd_api_requests_ongoing{entity_type="image"} 0
+lxd_api_requests_ongoing{entity_type="project"} 0
+lxd_api_requests_ongoing{entity_type="network"} 0
+lxd_api_requests_ongoing{entity_type="warning"} 0
+lxd_api_requests_ongoing{entity_type="profile"} 0
+lxd_api_requests_ongoing{entity_type="instance"} 0
+lxd_api_requests_ongoing{entity_type="identity"} 0
+lxd_api_requests_ongoing{entity_type="node"} 0
+lxd_api_requests_ongoing{entity_type="operation"} 0
 # HELP lxd_cpu_effective_total The total number of effective CPUs.
 # TYPE lxd_cpu_effective_total gauge
-lxd_cpu_effective_total{name="u1",project="default",type="container"} 4
-lxd_cpu_effective_total{name="vm",project="default",type="virtual-machine"} 0
-# HELP lxd_disk_read_bytes_total The total number of bytes read.
-# TYPE lxd_disk_read_bytes_total counter
-lxd_disk_read_bytes_total{device="loop5",name="u1",project="default",type="container"} 2048
```

Comment:
What happened to `lxd_disk_read_bytes_total` in the example?

Reply:
Same as the previous comment, I just wanted to make the example shorter without excluding non API metrics entirely. Since the example lenght is not a problem I will add these metrics back.